ci/requirements-2.7.pip

```diff
@@ -1,5 +1,7 @@
-httplib2
-google-api-python-client==1.2
-python-gflags==2.0
-oauth2client==1.5.0
+httplib2>=0.9.1
+google-api-python-client>=1.6.2, <2.0.0dev
```

Comment:
Yes, because if there is a 2.0 release by semantic versioning that is a breaking change.